README.md

```diff
@@ -103,7 +103,7 @@ sudo ./bin/linux_dependencies.sh
 ```
 If you're getting any error while running above command you need to install the following manually for your linux distro. : [adb](https://developer.android.com/studio/command-line/adb) [curl](https://curl.se/download.html) [tar]() [openjdk11]() [7zip](https://www.7-zip.org/download.html) [scrcpy](https://github.com/Genymobile/scrcpy)
 
-3. Run `wa_kdbe.py`
+3. Run `wa_kdbe.py`, double clicking the file works as well
```

Comment:
this only in mostly linux based system if you have given the path of interpreter in file itself. i.e. very first line of file is like 
```python
#!/bin/python
```
and have made you file executable by 
```shell
chmod +x filename.py
```
otherwise on windows only way I can think of this working with double click is if python in path and set as default app for .py files.
This would not work for everyone.